.github/workflows/release.yaml

```
@@ -58,7 +58,7 @@ jobs:
       - name: Upload MacOS build
         uses: actions/upload-artifact@v3
         with:
-          name: dependencies.sources-arm64
+          name: DependenciesSourceCode.zip
```

Is there any specific reason to change the file extension?

```suggestion
          name: DependenciesSourceCode.tar.gz
```

---

No specific reason. Updated.